FEATURE-LIFECYCLE.md

```diff
@@ -10,42 +10,42 @@ information at [Feature Lifecycle Checklist](FEATURE-LIFECYCLE-CHECKLIST.md).
 <table>
   <tr>
     <td></td>
-    <td>Development (pre-alpha)</td>
+    <td>Experimental</td>
     <td>Alpha</td>
     <td>Beta</td>
     <td>Stable</td>
   </tr>
   <tr>
     <td>Purpose</td>
-    <td>Active development of a feature, not for use by end users directly.</td>
+    <td> Feature is under active development and user facing APIs will most likely change. Users should deploy experimental features with extreme caution, preferably in non-production environments as newer versions can require a migration effort.</td>
```

Comment:
`most likely` -> `may`
`can` -> `may`
`newer versions` -> `experimental features` (you are talking about the feature status and then switch context to the version...